### DISCUSSION TITLE
2023-11-20-coffee

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#170] [General]
# 我的咖啡机入门实践 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life如何不到一千块钱实现每天好喝咖啡自由？想通过这篇短文来给大伙分享一下我的咖啡机购买使用以及配件选择的经验，属于最近买得最有生活意义的一个物件。

https://tw93.fun/2023-11-20/coffee.html

**Comment by aiguo688** [comment]
看来大佬是西安交大毕业的吧？

**Reply by tw93** [reply]
不是，我西电毕业的

**Comment by aiguo688** [comment]
咖啡豆也是在山姆购买的，冒昧的问一下，山姆是https://www.samsclub.cn/吗?

**Reply by tw93** [reply]
嗯嗯，不过我一般是线下的
